components/StyledComponents/StyledButton.js

```diff
@@ -3,13 +3,12 @@ import styled from 'styled-components';
 const StyledButton = styled.button`
   background-color: var(--secondary-color);
   border: none;
-  font-family: 'Inter', sans-serif;
+  font-family: var(--font-family);
   font-size: 20px;
   font-weight: 500;
   color: #ffffff;
-  border-radius: 50px;
+  border-radius: 999px;
```

Comment:
999 is probably too much.

* What is the expectation?
* Look for a better value here!

Reply:
here I want the button to be as round as possible. And I thought according to this it would be practical to set a high value to make sure that it looks the best on all screen sizes.

Reply:
You can try 50% bit that usually requires a square button (height === width)

If you want the button to have a consistent radius it should be half as big as the height of the button.  
I.e.
```css
button {
  height: 2rem;
  border-radius: 1rem; 
} 
```

Reply:
Example: https://codepen.io/dekk/pen/OJQZKBy/9bd9077a0417f2acd197a10ae01cba56